hypervisor/network/network_linux.go

```diff
@@ -790,7 +791,54 @@ func DeleteBridge(name string) error {
 	return nil
 }
 
-func AddToBridge(iface, master *net.Interface) error {
+// AddToBridge attch interface to the bridge,
+// we only support ovs bridge and linux bridge at present.
+func AddToBridge(iface, master *net.Interface, veth netlink.Link) error {
+	switch master.Name {
```

Comment:
We can not get the bridge name from master, master point to the ovs-system device, not the bridge we would like to attach to